clusterloader2/pkg/measurement/common/bundle/test_metrics.go

```diff
@@ -155,60 +156,60 @@ func (t *testMetrics) Execute(config *measurement.MeasurementConfig) ([]measurem
 	switch action {
 	case "start":
 		summary, err := execute(t.etcdMetrics, actionStartConfig)
-		appendResults(&summaries, errList, summary, err)
+		appendResults(&summaries, errList, summary, err, t.etcdMetrics.String(), action)
```

Comment:
Sorry for being pedantic, but for append shouldn't really do this kind of munging.

What about doing the following:
- create a function like:
```
func executeError(measurement, action string, err error) error {
  if err != nil {
    return goerrors.Errorf("action %s failed for %s measurement: %v", action, measurement, err)
  }
  return nil
}
```
- leaving appendResults as is
- changing the calls to e.g.
```
appendResults(&summaries, errList, summary, executeError(t.etcdMetrics.String(), action, err))
```

Reply:
sounds good, will do @wojtek-t